docs/API/InstrumentFormat.md

```diff
@@ -109,6 +143,15 @@ the following format:
 `Description`: The human readable description of this element, such
                as question or label text. This may or may not be required
                depending on element type.
+               
+`DisplayIf`: A formula that returns true when the element should be
+             displayed. Primarily used to display fields based on
+             candidate context or previous-answers. Defaults to `true`.
+             
+`HiddenSurvey`: A formula that returns true if the element should be hidden
+                from respondants of a survey. Defaults to `false`.
+                
+`Hidden`: A formula that returns true if the element should ALWAYS be hidden. Defaults to `false`.
```

Comment:
The word "ALWAYS" doesn't make sense in this sentence. If it should always be hidden, then it's not a formula, it's always hidden.

If this sentence is trying to say what I think, then I don't see the difference between `Hidden` and `DisplayIf`

Reply:
> I don't see the difference between Hidden and DisplayIf
You're right, it was a CAP requirement for a very specific edge case. Removing `Hidden` entirely would be the best move imo.